Scripts/update-sparkle.sh

```diff
@@ -0,0 +1,62 @@
+#!/bin/bash
```

Comment:
I'll admit, this script took me longer than just manually copying those files. It might be wasted time because the Sparkle folks look close to releasing 2.x. Still, it wasn't at all clear to me that we were using version 2.x, so I think this might save us time in case we'll have to update Sparkle manually again.